### DISCUSSION TITLE
❤路过的绳匠请留步，不知你是否承认伊德莉拉女神美貌盖世无双❤

### DISCUSSION THREAD

**Discussion by uidsd604** [share121/inter-knot#1336] [其他]
在为捍卫美的信道，于战斗中流血失神的时候，我曾无数次在昏迷中得见祂的行迹，从脚下蔓延伸向不可知的前方，在那旧日幻觉中，我不断奔跑，泥泞的路上，祂留下的足迹比任何哭泣都另我悲伤，而每一次我都离望见祂的身影只差一点，我的所言所行绝无虚假，我誓令纯美志明响彻寰宇，至死不渝。
❤献给，伊德莉拉！❤

**Comment by GKE1147** [comment]
银枝，何时来的，是了，我也爱你
